compositor_render/src/registry.rs

```diff
@@ -56,4 +64,14 @@ impl<T: Clone> RendererRegistry<T> {
 
         Ok(())
     }
+
+    pub(crate) fn unregister(&mut self, id: &RendererId) -> Result<(), UnregisterError> {
+        match self.registry.remove(id) {
+            Some(_) => Ok(()),
+            None => Err(UnregisterError {
+                item_type: self.registry_type.registry_item_name(),
+                renderer_id: id.clone(),
+            }),
+        }
```

Comment:
Use `.ok_or`

Reply:
ok_or maps None to Err, but here I need both map None to Err and Some(value) to `Ok(())`